### DISCUSSION TITLE
Welcome to toot-together Discussions!

### DISCUSSION THREAD

**Discussion by joschi** [joschi/toot-together#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gr2m** [comment]
Hey I just found out about your fork, very cool! Feel free to send a pull request to mention your Mastadon fork in the twitter-together README. You can add a section called "Similar projects" :)
